pkg/operator/rbaccontroller/rbaccontroller.go

```diff
@@ -0,0 +1,105 @@
+package rbaccontroller
+
+import (
+	"context"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	listerrbacv1 "k8s.io/client-go/listers/rbac/v1"
+	"k8s.io/klog"
+)
+
+const (
+	// If set to "true" for a cluster role binding, RBACController would remove unauthenticated
```

Comment:
As per a previous comment from @sttts, please indicate that this annotation is set by the operand e.g.

```
This annotation is set to a value of `true` on RBAC resources created by the openshift-apiserver.
```

Reply:
I see what is missing! updated the comment now.